tests/zenko_e2e/backbeat/ReplicationUtility.js

```diff
@@ -561,6 +563,57 @@ class ReplicationUtility {
             return cb();
         });
     }
+    
+    assertNoBlob(containerName, key, cb) {
+        this.azure.getBlobProperties(containerName, key, err => {
+            assert(err.code);
+            return cb();
+        });
+    }
+
+    checkReplicationStatus(locationName, cb) {
+        const hostname = 'zenko.local';
+        let path = '/_/backbeat/api/crr/status';
+        if (locationName) {
+            path = `${path}/${locationName}`;
+        }
+        makeGETRequest(path, (err, res) => {
+            getResponseBody(res, (err, body) => {
+                return cb(null, body);
+            });
+        });
+    }
+
+    checkReplicationResumeSchedule(locationName, cb) {
+        const path = `/_/backbeat/api/crr/resume/${locationName}`;
+        makeGETRequest(path, (err, res) => {
+            getResponseBody(res, (err, body) => {
+                return cb(null, body);
+            });
+        });
+    }
+
+    pauseReplication(locationName, cb) {
+        let path = '/_/backbeat/api/crr/pause';
+        if (locationName) {
+            path = `${path}/${locationName}`;
+        }
+        makePOSTRequest(path, '{}', cb);
+    }
+
+    resumeReplication(locationName, schedule, hoursScheduled, cb) {
+        const host = 'zenko.local';
```

Comment:
this is used a couple of times in different functions, could be pulled out and made global like `src` & `destLocation`

Reply:
this actually is not longer needed here, ty for noticing